packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -152,6 +152,7 @@ export type OutdatedCommandOptions = {
 | 'strictSsl'
 | 'tag'
 | 'userAgent'
+| 'userConfig'
```

Comment:
it will be easier if you make this new option optional. And it will be better because it won't be a breaking change.

Reply:
I thought about that, but that could mean that a certain feature could not support the token helper interface, because this option wasn't included. I hadn't considered it a breaking change, because I was seeing it from the point of view of the CLI. I hadn't considered from the API level - the `userConfig` for `@pnpm/client` is actually optional, I'm not sure how best to ensure this feature doesn't break in surprising ways for certain commands (due to missing passing on the `userConfig`).

If this were to be non-optional, should I break the commits up to which bits are breaking so that only the affected packages have breaking changes (sorry I'm not familiar with how the release process is built in this monorepo)

Reply:
make the tests pass, then I'll check